backend/src/main/java/com/crm/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package com.crm.security;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
```

Comment:
Move .authorizeRequests() to 1st line, i.e. http.authorizeRequests(). It looks better :)